src/orion/core/worker/producer.py

```diff
@@ -132,7 +132,7 @@ def register_trials(self, new_point):
             return 1
 
         except DuplicateKeyError:
-            log.debug("#### Duplicate sample.")
+            log.info("#### Duplicate sample.")
```

Comment:
I'm not sure this should be info. What do you think?

Reply:
Fixed.